src/components/PersonLink.tsx

```diff
@@ -0,0 +1,59 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const params = useParams();
+  const { sex, name, born, died, fatherName, motherName, slug } = person;
+
+  const mother = motherName ? people.find(p => p.name === motherName) : null;
+
+  const father = fatherName ? people.find(p => p.name === fatherName) : null;
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': params.slug === slug })}
```

Comment:
The use of `classnames` library is good for conditional class names. However, ensure that the condition is correct. The current condition checks if the `params.slug` is equal to the current person's `slug`. This might be a potential bug if the `params` object does not have a `slug` property or the routing parameters are not named `slug`. Double-check the route configuration to ensure that this is the correct parameter name.